src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -81,27 +84,38 @@ public EditCommand(Index index, EditStudentDescriptor editStudentDescriptor,
     @Override
     public CommandResult execute(Model model) throws CommandException {
```

Comment:
This method is kinda long now (~35 LOC), you might want to split it up into some helpers methods.

Since you've split the boolean declaration up, you could put those into a `private boolean isSameStudent()` and `private boolean hasClashingClassTime()` or something.